main.py

```diff
@@ -1,8 +1,26 @@
-import json
-import werkzeug
+import os, json, werkzeug
+import logging 
 from flask import Flask, request
+from werkzeug.security import generate_password_hash, check_password_hash
 
 
+logging.basicConfig(filename="main.log", 
+                    format='%(asctime)s %(message)s', 
+                    filemode='w') 
+logger = logging.getLogger() 
+logger.setLevel(logging.ERROR) 
+
+def load_credits(path_credit):
+
+    try:
+        with open(path_credit) as f:
+            credit = json.load(f)
+            return credit
+    except FileNotFoundError:
```

Comment:
You can capture the error and return it to the user as well. There are also better ways of valye printing:
```
except FileNotFoundError as err:
  logger.error(f'FileNotFoundError: no credentials.txt in {os.path.dirname(os.path.realpath(__file__))}: {err}')
```
This works since python 3.6.

Reply:
How then I should refer to err in test here:
`mock_error.assert_called_once_with(f'FileNotFoundError: no credentials.txt in {os.path.dirname(os.path.dirname(os.path.realpath(__file__)))}: {???}')`

Reply:
Suddenly, I start to realize that better use os.environ for defining path to credentials.txt. Am I right?